### ISSUE TITLE
PUSHER_CLIENT_ENCRYPTED is ignored when value is «false»

### ISSUE THREAD

**Issue by yivo** [openware/peatio#804]

**Comment by yivo** [comment]
The solution is:
```
    gon.pusher = {
      key:      ENV.fetch('PUSHER_CLIENT_KEY'),
      wsHost:   ENV.fetch('PUSHER_CLIENT_WS_HOST'),
      httpHost: ENV['PUSHER_CLIENT_HTTP_HOST'],
      wsPort:   ENV.fetch('PUSHER_CLIENT_WS_PORT'),
      wssPort:  ENV.fetch('PUSHER_CLIENT_WSS_PORT')
    }.reject { |k, v| v.blank? }
     .merge(encrypted: ENV.fetch('PUSHER_CLIENT_ENCRYPTED').present?)
```

**Comment by yivo** [comment]
Fixed in #811.
